src/data/firebase.js

```diff
@@ -0,0 +1,95 @@
+import { initializeApp } from 'firebase/app';
+import { getDatabase, ref, get } from 'firebase/database';
```

Comment:
ref as `createFirebase...`

Reply:
Now I don't  use ```ref``` method.
```suggestion
import {
  doc,
  addDoc,
  setDoc,
  getDoc,
  getFirestore,
  collection,
  getDocs,
  deleteDoc,
  query,
  where,
} from 'firebase/firestore';
```